src/swarm/util/Verify.d

```diff
@@ -0,0 +1,81 @@
+/*******************************************************************************
+
+    Helper module to use `verify` from ocean v3.4 instead of `assert`, even if
+    the ocean submodule is at a version that does not include
+    `ocean.core.Verify`. This is to allow for writing new swarm code that can be
+    safely merged into v5.
+
+    Copyright: Copyright (c) 2018 sociomantic labs GmbH. All rights reserved
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module swarm.util.Verify;
+
+import ocean.core.VersionCheck;
+
+static if (hasFeaturesFrom!("ocean", 3, 4)) public import ocean.core.Verify;
```

Comment:
Please, at least split the statement onto own line ;)

```D
static if (hasFeaturesFrom!("ocean", 3, 4))
    public import ocean.core.Verify;
else:
```

Reply:
ok, ok

Reply:
<3

Reply:
It is not a statement but an import declaration :P